services/web-app/data/carbon.yml

```diff
@@ -0,0 +1,279 @@
+designKits:
+  carbon-white-sketch:
+    name: White theme
+    tool: sketch
+    type: components
+    status: stable
+    url: sketch://add-library/cloud/557b75ff-67d3-41ab-ada5-fa25447218c1
+    action: link
+    sponsor: carbon
+    license: Apache 2.0
+  carbon-g10-sketch:
+    name: Gray 10 theme
+    tool: sketch
```

Comment:
if libraries are the entrypoint of data into the app (we use the contents of libraries.js to find the top-level carbon.yml which defines the library and then find all carbon.ymls in the subtree), how does this make it in if it's not associated to a lib? 🤔

Reply:
what would "design kits get indexed first" mean? that they'd be in a different dedicated file? or just we read design kits *first* from the carbon.yml? if the latter I think my question still stands

Reply:
I believe this question has been resolved through https://github.com/carbon-design-system/carbon-platform/issues/930.